### PR TITLE
Remove 'kubeadm alpha kubelet config download'

### PR DESCRIPTION
**Pull request by prasadkatti**
- Remove `kubeadm alpha kubelet config download` from 'kubeadm alpha' reference page.
- Fix the `kubeadm alpha kubelet config enable-dynamic` link to included page.
- Remove generated file `content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_kubelet_config_download.md`

Refer https://github.com/kubernetes/kubernetes/pull/87944

/sig cluster-lifecycle

### CODE REVIEW

**Comment by neolit123** [comment]
> Do I also need to remove the generated file?

yes, looks like the folder is not cleaned up on regeneration for new releases.

here:
https://github.com/kubernetes/website/tree/810fc2bb57817354dd4091338921f6f370ebacdf/content/en/docs/reference/setup-tools/kubeadm/generated
it shows that the file is outdated compared to the rest.

/sig cluster-lifecycle

**Comment by kbhawkey** [comment]
> /cc @tengqm @kbhawkey any thoughts on `generated/kubeadm_alpha_kubelet_config_download.md` getting left behind with reference generation?

Yes, if the generated file is no longer needed but still _included_ in the `kubeadm/kubeadm-alpha.md` page, then:

- remove the include statement from this file
- remove the previously generated file, `generated/kubeadm_alpha_kubelet_config_download.md`

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md [hunk 1]
```diff
@@ -65,7 +65,6 @@ to enable the DynamicKubeletConfiguration feature.
 
 {{< tabs name="tab-kubelet" >}}
 {{< tab name="kubelet" include="generated/kubeadm_alpha_kubelet.md" />}}
-{{< tab name="download" include="generated/kubeadm_alpha_kubelet_config_download.md" />}}
```
Comment (kbhawkey) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md:68]: @prasadkatti ,
If you have time, would you double check that "enable-dynamic" is including the correct file?
I see a generated file, `generated/kubeadm_alpha_kubelet_config_enable-dynamic.md`

Comment (prasadkatti) [reply content/en/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md:68]: You are right. It is including the wrong file. I can fix that.

**Comment by prasadkatti** [comment]
ready for another review

**Comment by kbhawkey** [comment]
Thanks @prasadkatti .
I guess the tab introductory text is correct,
`Use the following commands to either download the kubelet configuration from the cluster or to enable the DynamicKubeletConfiguration feature.`

Page preview:
https://deploy-preview-20513--kubernetes-io-master-staging.netlify.app/docs/reference/setup-tools/kubeadm/kubeadm-alpha/#cmd-phase-kubelet
/lgtm

**Comment by prasadkatti** [comment]
Oh. I can update the introductory text as part of this same PR.

**Comment by prasadkatti** [comment]
/assign @kbhawkey
@kbhawkey - I have updated the introductory text. PTAL.

**Pull request merged**